modules/gltf/test/lib/gltf-scenegraph-modifiers.spec.js

```diff
@@ -41,13 +42,12 @@ test('GLTFScenegraph#addImage', t => {
 });
 
 test('GLTFWriter#Should be able to write custom attribute', async t => {
-  const i3sContent = await loadI3STileContent();
+  const inputData = await load(GLTF_BINARY_URL, GLTFLoader, {gltf: {postProcess: true}});
   const gltfBuilder = new GLTFScenegraph();
 
-  i3sContent.attributes.positions.value = new Float32Array(i3sContent.attributes.positions.value);
   gltfBuilder.addMesh({
-    positions: i3sContent.attributes.positions,
-    _BATCHID: i3sContent.attributes.normals
+    positions: inputData.meshes[0].primitives[0].attributes.POSITION,
```

Comment:
I guess the standard attribute name for glTF is `POSITIONS` or `POSITION` (e.g. upper case) - we do convert but maybe "teach" the standard usage here?

Reply:
Done